### PR TITLE
play: 🐸 show correct quotation after ticker changed #847

### PR DESCRIPTION
**Pull request by arealclimber**
### DEVELOP

- [x] OpenPositionItem
- [x] OpenSubTab
- [x] PositionClosedModal

### CHECK LIST

- [x]  #847 
- [x] #877
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

File: src/components/position_closed_modal/position_closed_modal.tsx [hunk 1]
```diff
@@ -86,13 +86,13 @@ const PositionClosedModal = ({
...
 
   const spread = marketCtx.getTickerSpread(openCfdDetails.targetAsset);
-  const pnl = toPnl({
-    openPrice: openCfdDetails.openPrice,
-    closePrice: predictedClosePrice,
-    amount: openCfdDetails.amount,
-    typeOfPosition: openCfdDetails.typeOfPosition,
-    spread: spread,
-  });
+  // const pnl = toPnl({
```
Comment (arealclimber) [inline src/components/position_closed_modal/position_closed_modal.tsx:89]: Deprecated

File: src/components/position_closed_modal/position_closed_modal.tsx [hunk 2]
```diff
@@ -438,11 +452,14 @@ const PositionClosedModal = ({
           autoClose: false,
         });
         return;
-      }
-      const displayedCloseOrder = toDisplayCloseOrder(openCfdDetails, quotation);
-      globalCtx.dataPositionClosedModalHandler(displayedCloseOrder);
+      } else if (quotation.ticker.split('-')[0] === openCfdDetails.ticker) {
```
Comment (arealclimber) [inline src/components/position_closed_modal/position_closed_modal.tsx:455]: TODO: 在 Context 驗證簽名

**Review by TzuHanLiang — APPROVED** [review]
- checked

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**